app/models/work_view_stat.rb

```diff
@@ -0,0 +1,19 @@
+class WorkViewStat < ActiveRecord::Base
+  extend Sufia::WorkStatUtils
```

Comment:
For work statistics I've been following the pattern already in place for files statistics.